Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Spawning/DespawnNodeable.cpp

```diff
@@ -50,10 +50,11 @@ namespace ScriptCanvas::Nodeables::Spawning
         }
     }
 
-    void DespawnNodeable::RequestDespawn(SpawnTicketInstance spawnTicket)
+    void DespawnNodeable::RequestDespawn(AzFramework::EntitySpawnTicket spawnTicket)
     {
-        if (!spawnTicket.m_ticket)
+        if (!spawnTicket.IsValid())
         {
+            AZ_Error("DespawnNodeable", false, "EntitySpawnTicket is missing or invalid.")
```

Comment:
nit: It's not missing since its being accessed right now. Just 'invalid' is fine here I think

Reply:
I would also add a bit of text to make the description unique from other messages. This same message is used in other location and if users report that message it'll be more difficult to tell which of the exact calls were made. As a recommendation, consider writing something along the lines of "EntitySpawnTicket used for despawning is invalid".